src/schema-generation/flex-search-filter-input-types/filter-fields.ts

```diff
@@ -0,0 +1,232 @@
+import { getNamedType, GraphQLInputType, GraphQLList, GraphQLNonNull } from 'graphql';
+import * as pluralize from 'pluralize';
+import { isArray } from 'util';
+import { Field, FlexSearchLanguage, TypeKind } from '../../model';
+import { BinaryOperationQueryNode, BinaryOperator, ConstBoolQueryNode, FieldPathQueryNode, LiteralQueryNode, QueryNode, RuntimeErrorQueryNode } from '../../query-tree';
+import { FlexSearchFieldExistsQueryNode } from '../../query-tree/flex-search';
+import { AND_FILTER_FIELD, FILTER_FIELD_PREFIX_SEPARATOR, INPUT_FIELD_EQUAL, INPUT_FIELD_NOT, INPUT_FIELD_NOT_STARTS_WITH, INPUT_FIELD_STARTS_WITH, OR_FILTER_FIELD } from '../../schema/constants';
+import { AnyValue, PlainObject } from '../../utils/utils';
+import { QueryNodeResolveInfo } from '../query-node-object-type';
+import { TypedInputFieldBase } from '../typed-input-object-type';
+import { not } from '../utils/input-types';
+import { FlexSearchFilterObjectType } from './/generator';
+import { FLEX_SEARCH_FILTER_DESCRIPTIONS, FLEX_SEARCH_OPERATORS_WITH_LIST_OPERAND, STRING_TEXT_ANALYZER_FILTER_FIELDS } from './constants';
+
+const NESTED_FIELD_SUFFIX = 'Aggregation';
+
+export interface FlexSearchFilterField extends TypedInputFieldBase<FlexSearchFilterField> {
+    getFilterNode(sourceNode: QueryNode, filterValue: AnyValue, path: ReadonlyArray<Field>, info: QueryNodeResolveInfo): QueryNode
+}
+
+function getDescription({ operator, typeName, fieldName }: { operator: string | undefined, typeName: string, fieldName?: string, isAggregation?: boolean }) {
+    let descriptionTemplate = FLEX_SEARCH_FILTER_DESCRIPTIONS[operator || INPUT_FIELD_EQUAL];
+    if (typeof descriptionTemplate === 'object') {
+        descriptionTemplate = descriptionTemplate[typeName] || descriptionTemplate[''];
+    }
+    return descriptionTemplate ? descriptionTemplate.replace(/\$field/g, fieldName ? '`' + fieldName + '`' : 'the value') : undefined;
+}
+
+export class FlexSearchScalarOrEnumFieldFilterField implements FlexSearchFilterField {
+    public readonly inputType: GraphQLInputType;
+    public readonly description: string | undefined;
+
+    constructor(
+        public readonly field: Field,
+        public readonly resolveOperator: (fieldNode: QueryNode, valueNode: QueryNode, flexSearchLanguage?: FlexSearchLanguage) => QueryNode,
+        public readonly operatorPrefix: string | undefined,
+        baseInputType: GraphQLInputType,
+        public readonly flexSearchLanguage?: FlexSearchLanguage
+    ) {
+        this.inputType = FLEX_SEARCH_OPERATORS_WITH_LIST_OPERAND.includes(operatorPrefix || '') ? new GraphQLList(new GraphQLNonNull(baseInputType)) : baseInputType;
+        this.description = getDescription({ operator: operatorPrefix, fieldName: field.name, typeName: field.type.name, isAggregation: this.operatorPrefix != undefined });
+
+        if (this.field.description) {
+            this.description = (this.description ? this.description + '\n\n' : '') + this.field.description;
```

Comment:
I think this is the place where too many `\n\n`s are generated if the field description is not set